src/wiki/urls.py

```diff
@@ -247,6 +253,11 @@ def get_pattern(app_name="wiki", namespace="wiki", url_config_class=None):
        single Django project.
        https://docs.djangoproject.com/en/dev/topics/http/urls/#topics-http-reversing-url-namespaces
     """
+    import warnings
+    warnings.warn(
+        "wiki.urls.get_pattern is deprecated and will be removed in next version, just include('wiki.urls')",
+        DeprecationWarning
+    )
     if url_config_class is None:
         url_config_classname = getattr(settings, 'URL_CONFIG_CLASS', None)
```

Comment:
This setting is also deprecated, good idea to state that in `wiki.conf.settings` in the docstring of `URL_CONFIG_CLASS`